projects/aca-content/src/lib/components/details/details.component.ts

```diff
@@ -59,12 +61,22 @@ import { CommentsTabComponent } from '../info-drawer/comments-tab/comments-tab.c
   encapsulation: ViewEncapsulation.None
 })
 export class DetailsComponent extends PageComponent implements OnInit, OnDestroy {
+  @Input()
+  readOnly = false;
```

Comment:
Is that used somewhere? I can't find usages

Reply:
I removed